packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.js

```diff
@@ -0,0 +1,105 @@
+import { LightningElement, api } from 'lwc';
+import thankYou from '@salesforce/label/c.cookbook_ThankYou';
+import provideDetails from '@salesforce/label/c.cookbook_ProvideDetails';
+import provideMoreDetails from '@salesforce/label/c.cookbook_ProvideMoreDetails';
+import dontKnowWhatToWrite from '@salesforce/label/c.cookbook_DontKnowWhatToWrite';
+
+
+/**
+ * The `descriptionStrengthIndicator` component is a dynamic indicator that shows the user if their case description has enough details so far. It also give hints to the user as to what information they show include in the description. 
+ * @example
+ * <c-description-strength-indicator></c-description-strength-indicator>
+ */
+export default class DescriptionStrengthIndicator extends LightningElement {
+  labels = {
+    thankYou,
+    provideDetails,
+    provideMoreDetails,
+    dontKnowWhatToWrite
+  }
+  
+  /**
+   * The label to be shown to the user.
+   * @api
```

Comment:
These @api tags are custom tags for our auto-generated doc so they're likely wasted here 😅